hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/OzoneClientProducer.java

```diff
@@ -102,12 +103,12 @@ public S3Auth getSignature() {
           signatureInfo.getSignature(),
           awsAccessId);
     } catch (OS3Exception ex) {
-      LOG.debug("Error during Client Creation: ", ex);
+      LOG.error("Error during Client Creation: ", ex);
```

Comment:
This should not be logged at `error`, a badly configured client can overwhelm the log.

Reply:
@kerneltime Thanks for the review.

Normally we'd expect the badly configured client to correct their configurations after they received the OS3Exception. But if the client doesn't change and keeps sending requests with bad configuraitons, the server might be in a blind state. With some explicit log, server-side can notice the bad operations and remind the client to correct their operations.

Reply:
I agree with @kerneltime: client-side problem should not be logged as error at server-side.  Client receives the error anyway and can act on it.

Reply:
Ok, reverted debug log for OS3Exception.